lib/Doctrine/DBAL/Types/TimeImmutableType.php

```diff
@@ -12,14 +12,18 @@ class TimeImmutableType extends TimeType
 {
     /**
      * {@inheritdoc}
+     *
+     * @return string
      */
-    public function getName()
+    public function getName() : string
     {
         return Type::TIME_IMMUTABLE;
     }
 
     /**
      * {@inheritdoc}
+     *
+     * @return string|false|null
```

Comment:
`false` should be eliminated.

Reply:
It was probably inferred from `DateTimeImmutable::format()`.